config/routes.rb

```diff
@@ -21,7 +21,7 @@
     delete '/delete_user/:id', to: 'update_user#destroy_user', as: :delete_user
   end
 
-  resources :users, only: [:edit, :update, :create]
+  resources :users, only: [:edit, :update, :create, :show]
```

Comment:
I'm going back and forth about the naming here. We have so many different types of users and different kind of pages that I'm worried UsersController/user_path/user-show might get confusing especially compared to the requester profile (a user_show of sorts) and professional profile.   My counter argument to myself is that it does make a lot of sense to have a users#update be what updates the user.  
Thoughts? @alishersadikov @rtravitz @meganft